src/platform/silabs/wifi/wfx_host_events.h

```diff
@@ -94,34 +95,52 @@ typedef struct __attribute__((__packed__)) sl_wfx_mib_req_s
     sl_wfx_mib_req_body_t body;
 } sl_wfx_mib_req_t;
 
-#else /* End WF200 else RS911x */
+#else /* End WF200 else RS9116,917 NCP and 917 SoC */
 
 #include "wfx_msgs.h"
 
 #if (SLI_SI91X_MCU_INTERFACE | EXP_BOARD)
 #include "sl_si91x_types.h"
 #include "sl_status.h"
 #include "sl_wifi_constants.h"
+#include "stdbool.h"
```

Comment:
```suggestion
#include <stdbool.h>
```
are we using some internal stdbool header ?

Reply:
done @rosahay-silabs